addons/utm/views/utm.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
-        <menuitem id="menu_link_tracker_root" name="Link Tracker" groups="base.group_no_one" sequence="30"/>
+        <menuitem id="menu_link_tracker_root" name="Link Tracker" groups="base.group_no_one" sequence="30" web_icon="utm,static/description/icon.png"/>
```

Comment:
It's not an application (It's even a hidden module).

Why should we add an icon ?

Reply:
Its written in specification, 
Set a real icon to app "utm" since it shows up in homepage debug mode.